students/yushus/session02/print_grid/print_grid.py

```diff
@@ -0,0 +1,26 @@
+# control the height or how many rows to print
+for l in range(2):
+    # print the first + sign
+    print('+', end='')
+
+    # control the length or how many columns to print
+    for i in range(2):
+        # one iteration looks like: ----+
+        print('-'*4,end='')
```

Comment:
Minor: I think there's a blank space for each `-` character printed, but I'd say it's not a big deal. You may want to compare your outputs with the exact printouts from the example outputs.

Reply:
I see. Included the space after each -